packages/next/build/profiler/profiler.js

```diff
@@ -0,0 +1,148 @@
+import path from 'path'
+import mkdirp from 'mkdirp'
+import fs from 'fs'
+
+let inspector
+try {
+  // eslint-disable-next-line node/no-unsupported-features/node-builtins
+  inspector = require('inspector')
+} catch (e) {
+  console.log('Unable to CPU profile in < node 8.0')
+}
```

Comment:
Is this `try/catch` needed since our minimum supported node version is v8?

Reply:
It's not needed, but I tried to leave as much as possible of the original file.